modules/gltf/src/gltf/gltf-instantiator.ts

```diff
@@ -199,16 +200,6 @@ export class GLTFInstantiator {
   }
 }
 
-enum GL {
```

Comment:
Nit: I understand the wish to avoid redefinition, however in v9, my soft preference is that the constants are only used in the webgl module. 

While the inliner will technically strip them in the end it is quite heavy to include them here in a module which is supposed to be webgl independent. If we start adding more constant, then it makes more sense to import.

Reply:
That's helpful context -- thanks! I've reverted the change, and removed another dependency on `@luma.gl/constants` elsewhere in this module.

Does that imply that `@luma.gl/constants` should *only* contain WebGL-specific constants? I see that's true now, just wasn't sure if that was the long-term plan.

Note that because we're declaring `GL` locally this way, the inliner will _not_ strip these constants, doing so would require the earlier `EXCLUDED_TYPES` approach we'd discussed. Seems fine to me if we don't inline them here, since (1) these are only a couple constants, and (2) this might be temporary anyway as we try to make this code backend-agnostic.

Reply:
Thanks, yes, I prefer loosely coupled modules even at a small cost.

We could skip the enum and declare them as on separate lines as `const GL_LINES = ...;` etc, then the minifier will get to them.